lib/molecules/menu/template.html

```diff
@@ -1,13 +1,13 @@
 <ul ng-if="lnItems.length">
   <li ng-repeat="item in lnItems" ng-class="item.class">
-    <a ng-href="{{item.link}}" ng-attr-target="{{item.new_window ? '_blank' : ''}}" >
+    <a ng-href="{{item.link}}" ng-class="{{lnParentClass}}" ng-attr-target="{{item.new_window ? '_blank' : ''}}" >
       {{item.title}}
       <span ng-if="item.items.length" ng-class="lnParentIconClass"></span>
     </a>
     <ul ng-if="item.items.length">
       <li ng-repeat="subItem in item.items" ng-class="subItem.class">
-        <a ng-href="{{subItem.link}}" ng-attr-target="{{subItem.new_window ? '_blank' : ''}}">{{subItem.title}}</a>
+        <a ng-href="{{subItem.link}}" ng-class="{{lnChildClass}}" ng-attr-target="{{subItem.new_window ? '_blank' : ''}}">{{subItem.title}}</a>
```

Comment:
Same as above, but considering the other comment about the name of this parameter, this should be `ng-class="lnItemClass"`.